src/main/java/org/springframework/data/hazelcast/repository/support/HazelcastEntityInformation.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2014-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.data.hazelcast.repository.support;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mapping.PersistentEntity;
+import org.springframework.data.mapping.model.MappingException;
+import org.springframework.data.repository.core.support.PersistentEntityInformation;
+
+import java.io.Serializable;
+
+/**
+ * <P>
```

Comment:
Use the Javadoc convention [1][2].  Meaning, the javadoc for a class should start from a sentence, not <P>. 

[1] https://google.github.io/styleguide/javaguide.html#s7-javadoc
[2] http://www.oracle.com/technetwork/articles/java/index-137868.html

Reply:
Done.